nf_core/launch.py

```diff
@@ -2,6 +2,7 @@
 """ Launch a pipeline, interactively collecting params """
 
 from __future__ import print_function
+from questionary import constants
```

Comment:
Is it possible to use `questionary.constants` instead of doing this import? I generally prefer this where possible as I think it makes the code easier to understand for newcomers.

Reply:
Oh snap, this should not be in there anymore. We don't need to import constants at all.
The questionary.constants file contains the selection errors, which I wanted to change to the one that PyInquirer used, which didn't work somehow.
Forgot to delete the import I guess ... actually I'm pretty sure I deleted it but must have slipped through anyway.

Reply:
```suggestion
```